### PR TITLE
feat(components/ag-grid): add hover style for header resize

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2856652](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2856652)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2119/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2119/storybooks/ag-grid-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2119/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/2119/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2119/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2119/playground/)

**Comment by Blackbaud-MattGregg** [comment]
@johnhwhite One issue in the spec we had was the color referenced for the gray divider line. It needs to be neutral-medium-dark for the line to meet color contrast requirement. It's also dotted. thx.

**Comment by Blackbaud-MattGregg** [comment]
@johnhwhite Update looks great! Slick. thx.

File: libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html [hunk 1]
```diff
@@ -44,7 +43,6 @@
     [data]="skyComponentProperties?.data"
     [debounceTime]="skyComponentProperties?.debounceTime"
     [descriptorProperty]="skyComponentProperties?.descriptorProperty"
-    [disabled]="skyComponentProperties?.disabled || false"
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html:47]: why remove this?

File: libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts [hunk 2]
```diff
@@ -292,6 +295,19 @@ export class SkyAgGridService implements OnDestroy {
       [SkyCellClass.Invalid]: getValidatorFn(),
     };
 
+    function getHeaderClass(
+      ...classNames: string[]
+    ): (params: HeaderClassParams) => string | string[] | undefined {
+      return (params: HeaderClassParams): string | string[] | undefined => {
+        const minWidth = Number(params.column?.getMinWidth());
+        const maxWidth = Number(params.column?.getMaxWidth());
+        if (params.column?.isResizable() && minWidth < maxWidth) {
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts:306]: It looks like further down we define `minWidth` and `maxWidth` as undefined by default. `undefined < undefined` is false. Does this mean that in order to get the resizing styling consumers must set min and max width?

File: libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html (hunk 1 above)
Comment (johnhwhite) [reply libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.html:47]: This is a reactive form so it should not be passing this input. It's handling the form control enable/disable within the component `.ts` file.

File: libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts (hunk 2 above)
Comment (johnhwhite) [reply libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts:306]: Added `??` operator values.

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Pull request merged**